core/utils/silly-i18n/src/index.js

```diff
@@ -90,10 +43,19 @@ export class I18n {
                     " is not a key in the translations hash provided"
             );
         }
+
         this.language = language in translations ? language : fallbackLanguage;
         this._translations = translations[this.language];
     }
 
+    /**
+     * Know which sections are present in the original hash
```

Comment:
Who knows that, the function? :D Also, does it indeed return "the translated string"? I'm not sure what it does based on the comment. Looking at the code, I would think it's something like: `List all sections` or something?

Reply:
It's just copypasta...